test/support/integration/teams_server.ex

```diff
@@ -253,13 +245,16 @@ defmodule Livebook.TeamsServer do
   end
 
   defp env(app_port, state_env) do
-    env = %{
-      "MIX_ENV" => "livebook",
-      "PORT" => to_string(app_port),
-      "DEBUG" => debug()
-    }
-
-    env = if proto(), do: Map.merge(env, %{"LIVEBOOK_PROTO_PATH" => proto()}), else: env
+    env =
+      :maps.filter(
+        fn _key, value -> value not in ["", nil] end,
```

Comment:
There is `Map.filter/2`, but in this case I would do similar to before like this:

```elixir
env =
  if path = System.get_env("TEAMS_LIVEBOOK_PROTO_PATH") do
    Map.put(env, "LIVEBOOK_PROTO_PATH")
  else
    env
  end
```